### PR TITLE
Distribution getprobabilities

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: lib/src/Uncertainty/Model/DistributionImplementation.cxx [hunk 1]
```diff
@@ -2458,6 +2458,15 @@ NumericalSample DistributionImplementation::getSupport() const
   return getSupport(getRange());
 }
 
+/* Get the discrete probability levels */
+NumericalPoint DistributionImplementation::getProbabilities() const
+{
+  if (!isDiscrete())
+    throw InternalException(HERE) << "Error: cannot return probability levels of a non discrete distribution.";
+
+  return computeCDF(getSupport()).getImplementation()->getData();
```
Comment (adutfoy) [inline lib/src/Uncertainty/Model/DistributionImplementation.cxx]: -1 It is the PDF which gives the probability to take a given value for a discrete random vector, not the CDF

**Comment by regislebrun** [comment]
No chance to get the C++ tests updated? There are still situations in which python tests fail due to a bug in the swig layer, and a C++ test is very useful to detect these situations.

**Comment by regislebrun** [comment]
Otherwise the PR is ok.

**Comment by jschueller** [comment]
@regislebrun, tested in c++ too.

**Comment by regislebrun** [comment]
+1

**Comment by sofianehaddad** [comment]
+1

**Comment by jschueller** [comment]
Turns out I dont need it for otrobopt.

**Comment by regislebrun** [comment]
But this service is useful even if it is not needed by your development! It is the natural counterpart of getSupport(), why not add it to the platform?

**Pull request merged**